src/site/content/en/blog/centering-in-css/index.md

```diff
@@ -0,0 +1,347 @@
+
+{% Aside "key-term" %} Macro layouts are like states or territories of a
+  country, very high level, large coverage zones. The zones created by macro
+  layouts tend to contain more layouts. The less surface the layout covers, the
+  less of a macro layout it becomes. As a layout covers less surface area or
+  contains less layouts, it becomes more of a micro layout. {% endAside %}
+
+### Autobot
+
+<figure class="w-figure w-figure--fullbleed">
+  <video playsinline controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/atoms-sandbox.google.com.a.appspot.com/autobot-ringer-cycle.mp4">
+  </video>
+</figure> 
+
+1. **Squish:** great
+2. **Squash:** great
+3. **Duplicate:** fine
+4. **Edit:** great
+5. **Flow:** great
+
+The container is set to flex with no alignment styles, while the direct children
+are styled with auto margins. There's something nostalgic and wonderful about
+`margin: auto` working on all sides of the element.
+
+```css
+.autobot {
+  display: flex;
+}
+.autobot > * {
+  margin: auto;
+}
+```
+
+<div class="w-columns">
+{% Compare 'worse', 'Pros' %}
+- Fun trick
+- Quick and dirty
+
+{% endCompare %}
+
+{% Compare 'better', 'Cons' %}
+- Awkward results when overflowing
+- Reliance on distribution instead of gap means layouts can occur with children
+  touching sides
+- Being "pushed" into position doesn't seem optimal and can result in a change
+  to the child's box size
+
+{% endCompare %}
+</div>
+
+**Great for** centering icons or pseudo-elements.
+
+### Fluffy Center
+
+<figure class="w-figure w-figure--fullbleed">
+  <video playsinline controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/atoms-sandbox.google.com.a.appspot.com/fluffy-center-ringer-cycle.mp4">
+  </video>
+</figure> 
+
+1. **Squish:** bad
+2. **Squash:** bad
+3. **Duplicate:** bad
+4. **Edit:** great!
+5. **Flow:** great! (so long as you use logical properties)
+
+Contestant "fluffy center" is by far our tastiest sounding, and is the only
+centering technique that's entirely element/child owned. See our solo inner pink
+border!? 
+
+```css
+.fluffy-center {
+  padding: 10ch;
+}
+```
+
+<div class="w-columns">
+{% Compare 'worse', 'Pros' %}
+- Protects content
+- Atomic
+- Every test is secretly containing this centering strategy
+- Word space is gap
+
+{% endCompare %}
+
+{% Compare 'better', 'Cons' %}
+- Illusion of not being useful
+- There's a clash between the container and the items, naturally since each are
+  being very firm about their sizing
+
+{% endCompare %}
+</div>
+
+**Great for** word or phrase-centric centering, tags, pills, buttons, chips, and
+more.
+
+### Pop & Plop
+
+<figure class="w-figure w-figure--fullbleed">
+  <video playsinline controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/atoms-sandbox.google.com.a.appspot.com/popnplop-ringer-cycle.mp4">
+  </video>
+</figure> 
```

Comment:
Sorry to make you re-do a video but we need to avoid the word `crazy` which is an offensive word to some (there's a long history of people in power labeling troublemakers and people they disapprove of as `crazy`)

Reply:
the full sentence doesnt mitigate the usage? "chaos of content muahahahaha crazy"?

i can reshoot, not a prob. lemme know if this subject in the sentence changes the usage. i understand `crazy` can be an offensive word, but i do want to learn more about when words like these words are ok too.

Reply:
I'm not an expert in this area, but my sense is that a lot of modern lingo is derived from ableist language. For example, saying something is "lame" is a way of saying that something doesn't meet expectations, but it's deriving that meaning from the assertion that someone with a range of movement impairment doesn't live up to some predefined standard. The same is true for words like crazy, dumb, stupid, etc. Those are all words originally created to refer to disabilities that then pivoted into the modern lexicon.

My rule of thumb is to try to never use these words and to look for alternatives.

Reply:
gotcha, so there's a list of words that you just essentially remove from your vocabulary entirely without exception. i'll follow suit.

Reply:
yeah pretty much. It can be a bit of a challenge to come up with good alternatives but sometimes it's fun :)

Reply:
video updated on gcp, may need to clear cache to get the new video from the cdn, here's a preview! 
![image](https://user-images.githubusercontent.com/1134620/93535060-0c982a00-f8fb-11ea-949d-985de528312b.png)